Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
+    cmds:
+      - >
+        go run github.com/air-verse/air@v1.61.1 \
+        --build.cmd "go build -o tmp/bin/main" --build.bin "tmp/bin/main" --build.delay "100" \
+        --build.exclude_dir "node_modules" \
+        --build.include_ext "go" \
+        --misc.clean_on_exit true
+    requires:
+      vars: [GITHUB_TOKEN, OPENAI_API_KEY]
+  test:
+    cmds:
```

Comment:
While using `go test -v ./...` is valid, consider adding an option to run tests with coverage or other flags that might be useful for your team. This could help improve the testing process and provide more insights during test runs.